include/irr/asset/IMeshManipulator.h

```diff
@@ -368,9 +377,9 @@ namespace asset
 		}
 		
 		//!
-		static void loadNormalQuantCacheFromBuffer(E_QUANT_NORM_CACHE_TYPE type, asset::ICPUBuffer* buffer);
+		static bool loadNormalQuantCacheFromBuffer(E_QUANT_NORM_CACHE_TYPE type, SBufferRange<ICPUBuffer>& buffer, CQuantNormalCache& quantNormalCache);
 
-		//!
+		//! idk if there is need for this function ?\_(?)_/?
```

Comment:
maybe make a boolean/flag in `loadNormalQuantCacheFromBuffer` whether to replace or merge/add the contents